### PR TITLE
Migrate 2600 BCE Edwin Smith Papyrus

### PR DESCRIPTION
**Pull request by Ismael-KG**
First migration with [new yaml](https://github.com/Ismael-KG/A-History-of-Research-Ethics/issues/37#issuecomment-987863310).

### CODE REVIEW

**Comment by Ismael-KG** [comment]
Hm, shouldn't I be able to see the Edwin Smith Papyrus story here? https://deploy-preview-6--zealous-lalande-f62e6f.netlify.app/migration

**Comment by Ismael-KG** [comment]
Under the "Checks" tab of this PR, we read:

> stories/bce 2600 edwin smith surgical papyrus.html - [view](https://deploy-preview-6--zealous-lalande-f62e6f.netlify.app/stories/bce 2600 edwin smith surgical papyrus)

That link takes you to an error page

File: Stories/BCE 2600 Edwin Smith Surgical Papyrus.md [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
+  bce: true
+  year: 2600
+  source: https://doi.org/10.1038/sc.1988.15
+  author: Hughes
+  source-date: 1988
+category: Improving Research
+tags: ["WhatWeValue"]
+media:
+  source: https://ars.els-cdn.com/content/image/1-s2.0-S0741521414008659-gr1.jpg
+  caption: Column from the Edwin Smith Papyrus displaying the hieratic script. Case 33, toward the bottom, presents a sternoclavicular dislocation and describes two vessels in the upper chest carrying blood to the respiratory tract. The background is brown and the text is sometimes red and sometimes black Source: https://www.sciencedirect.
```
Comment (yochannah) [inline Stories/BCE 2600 Edwin Smith Surgical Papyrus.md]:
```suggestion
  caption: "Column from the Edwin Smith Papyrus displaying the hieratic script. Case 33, toward the bottom, presents a sternoclavicular dislocation and describes two vessels in the upper chest carrying blood to the respiratory tract. The background is brown and the text is sometimes red and sometimes black Source: https://www.sciencedirect."
```

Try this to fix the build error for this page - there'sa colon in the main text body, and the yaml parser is confused and frightened about what to do with two colons on the same line.

**Review by yochannah — CHANGES_REQUESTED** [review]

**Comment by Ismael-KG** [comment]
IT WORKED I AM ADDING "" TO THE CAPTION BIT [HERE](https://github.com/Ismael-KG/A-History-of-Research-Ethics/issues/37#issuecomment-987863310) WHAT MAGIC

**Comment by Ismael-KG** [comment]
whoops, got too excited and accidentally closed this

**Pull request merged**